lib/dirac_coarse.cpp

```diff
@@ -56,11 +56,14 @@ namespace quda {
   void DiracCoarse::createY(bool gpu, bool mapped) const
   {
     int ndim = transfer->Vectors().Ndim();
+    if (ndim == 5 && transfer->Vectors().Nspin() != 4)
```

Comment:
When we re-add multi-RHS operator support we will be ripping out the old 5th dimensional trick.  With this in mind, and given this pull be merged then, can you add a comment to each line like this that we will need to update when we do so.  E.g., add `// FIXME MRHS NDIM hack` that we can easily grep.

Reply:
Done.